src/download-artifact.ts

```diff
@@ -6,10 +6,21 @@ async function run(): Promise<void> {
   try {
     const name = core.getInput(Inputs.Name, {required: false})
     const path = core.getInput(Inputs.Path, {required: false})
+    const artifactFolder = core.getInput(Inputs.ArtifactFolder, {
+      required: false
```

Comment:
minor, but you don't need to specify `required: false` since that's the default.